addons/web/static/src/xml/file_upload_progress_bar.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+
+<div t-name="web.ProgressBar" class="o_file_upload_progress_bar_wrapper">
```

Comment:
I think it's best to name class of `web.ProgressBar` as `o_file_upload_progress_bar` (no `wrapper`).
The inner part can be named `o_file_upload_progress_bar_value`.

jQuery names the widget `ui-progressbar` and the inner part is `ui-progressbar-value`, so this is somehow the same naming convention.